content/de/docs/setup/_index.md

```diff
@@ -7,76 +7,96 @@ content_template: templates/concept
-* Mehr operative Verantwortung übernehmen Möchten
-
-Wählen Sie eine [schlüsselfertige Cloud-Lösung](/de/docs/setup/) aus.
-
-## Schlüsselfertige On-Premises-Lösungen
-
-Mit diesen Lösungen können Sie Kubernetes-Cluster mit nur wenigen Befehlen in Ihrem internen, sicheren Cloud-Netzwerk erstellen.
-
-Sie sollten eine schlüsselfertige lokal betriebene Cloud-Lösung auswählen, wenn Sie:
-
-* Cluster in Ihrem privaten Cloud-Netzwerk bereitstellen möchten
-* Ein engagiertes SRE-Team haben
-* Über die Ressourcen zum Hosten und Überwachen Ihrer Cluster verfügen
-
-Wählen Sie eine [schlüsselfertige On-Premises-Lösung](/de/docs/setup/) aus.
-
-## Individuelle Lösungen
-
-Individuelle Lösungen geben Ihnen die größte Freiheit in Ihren Clustern, erfordern jedoch das meiste Know-how.
-Diese Lösungen reichen von Bare-Metal-Anbietern bis hin zu Cloud-Anbietern mit unterschiedlichen Betriebssystemen.
+{{% /capture %}}
 
-Wählen Sie eine [individuelle Lösung](/de/docs/setup/) aus.
+{{% capture body %}}
 
-{{% /capture %}}
+## Lernumgebung
+
+Benutzen Sie eine Docker-basierende Lösung, wenn Sie Kubernetes erlernen wollen: Von der Kubernetes-Community unterstützte Werkzeuge oder Werkzeuge in einem Ökosystem zum Einrichten eines Kubernetes-Clusters auf einer lokalen Maschine.
+
+{{< table caption="Local machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
+
+|Community           |Ökosystem     |
+| ------------       | --------     |
+| [Minikube](/docs/setup/learning-environment/minikube/) | [CDK on LXD](https://www.ubuntu.com/kubernetes/docs/install-local) |
+| [kind (Kubernetes IN Docker)](https://github.com/kubernetes-sigs/kind) | [Docker Desktop](https://www.docker.com/products/docker-desktop)|
+|                     | [Minishift](https://docs.okd.io/latest/minishift/)|
+|                     | [MicroK8s](https://microk8s.io/)|
+|                     | [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) |
+|                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
+|                     | [k3s](https://k3s.io)|
+
+
+## Produktionsumgebung
+
+Überlegen Sie sich bei der Bewertung einer Lösung für eine Produktionsumgebung, welche Aspekte des Betriebs eines Kubernetes-Clusters (oder von _abstractions_) Sie selbst verwalten oder an einen Anbieter auslagern möchten.
+
+Einige mögliche Abstraktionen von Kubernetes-Clustern sind  {{< glossary_tooltip text="applications" term_id="applications" >}}, {{< glossary_tooltip text="data plane" term_id="data-plane" >}}, {{< glossary_tooltip text="control plane" term_id="control-plane" >}}, {{< glossary_tooltip text="cluster infrastructure" term_id="cluster-infrastructure" >}} und {{< glossary_tooltip text="cluster operations" term_id="cluster-operations" >}}.
```

Comment:
The tooltip display tet is not translated here. Is that intentional?

Reply:
related to the svg description in English, so the user can easier find the connections